src/Adapter/Order/CommandHandler/DeleteProductFromOrderHandler.php

```diff
@@ -70,6 +100,49 @@ public function handle(DeleteProductFromOrderCommand $command)
         $order = $order->refreshShippingCost();
 
         Hook::exec('actionOrderEdited', ['order' => $order]);
+
+        $this->contextStateManager->restoreContext();
+    }
+
+    /**
+     * @param Order $order
+     * @param OrderDetail $orderDetail
+     * @param Cart $cart
+     *
+     * @return bool
+     */
+    private function updateCart(Order $order, OrderDetail $orderDetail, Cart &$cart)
+    {
+        $oldCartRules = $cart->getCartRules();
+        // Update Product Quantity in the cart
+        $return = $cart->updateQty(
+            $orderDetail->product_quantity,
+            $orderDetail->product_id,
+            $orderDetail->product_attribute_id,
+            false,
+            'down'
```

Comment:
Can we put the operators in constants ? (not mandatory for this PR)

Reply:
It's legacy... so I can't do this... (may be in an another PR :wink:)